lib/pronto/bundler_audit.rb

```diff
@@ -8,16 +8,40 @@ module Pronto
   # Pronto::BundlerAudit is a Pronto::Runner that:
   # 1. Updates the Ruby Advisory Database,
   # 2. Runs bundle-audit to scan the Gemfile.lock, and then
-  # 4. Returns an Array of Pronto::Message objects if any issues or advisories
+  # 3. Returns an Array of Pronto::Message objects if any issues or advisories
   # are found.
   class BundlerAudit < Runner
     GEMFILE_LOCK_FILENAME = "Gemfile.lock"
 
     # @return [Array<Pronto::Message>] per Pronto expectation
     def run
-      auditor = Auditor.new
-      auditor.call
+      return [] unless (results = Auditor.new.call) && results.size > 0
+
+      results.map do |result|
+        Message.new(GEMFILE_LOCK_FILENAME,
```

Comment:
Is this a `::Pronto::Message`? If so make it `::Pronto::Message.new`, please, to remove uncertainty. The namespaces here get confusing.